packages/synapse-react-client/src/components/HeaderCard.tsx

```diff
@@ -62,6 +66,11 @@ function HeaderCard({
     }
   })
 
+  const { appId } = useSynapseContext()
```

Comment:
Nick recommends changing this to be an optional prop passed into CardContainerLogic (and down eventually to the HeaderCard)

Reply:
This will allow us to roll this out smoothly to other portals (and potentially extend)